scripts/inspect.sh

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env bash
+
+INSPECT_DUMP=${SNAP_DATA}/inspection-report
+RETURN_CODE=0
+
+function print_help {
+  # Print the help message
+  printf -- 'This script will inspect your microk8s installation. It will report any issue it may find,\n';
+  printf -- 'and prepare tarball with logs and traces you can attach on issues you open against\n';
```

Comment:
"and create a tarball of logs and traces which can be attached to an issue filed against"

Reply:
Easy